project.yml

```diff
@@ -24,7 +27,9 @@ targetTemplates:
             - Resources
             - Config/${enviroment}
         scheme: {}
-        dependencies: []
+        dependencies:
```

Comment:
Instead of kingfished use AsyncImage which is available in SwiftUI from iOS15.

https://www.hackingwithswift.com/quick-start/swiftui/how-to-load-a-remote-image-from-a-url